### ISSUE TITLE
Critical Bug on Strength Challenge

### ISSUE THREAD

**Issue by Vicorin** [Lirin111/HandOfFate2-AccessibilityMod#3]
At the end of the second level of Strength, dialogue with the blacksmith is not advancing. The screen-reader is getting out of sync with what is there visually. Screen-reader can progress through the dialogue until the blacksmith says, "I admit I'm used to horseshoes, so this might take a while." From there, it just repeats the same text over and over, not letting you proceed to select an item. On screen, the text does not progress past the first bit of dialogue, which is why I think it might be a bug caused by the mod.

I tried restarting the game, restarting the run, going to a different encounter first, going to the inventory and coming back, with and without a controller connected. It let me through once, but I have not been able to get through a second time, which is odd. It happens even when I know I have a helmet, armor, and/or shield in my inventory.

Let me know if there's anything else I can do to help, and thanks for what you do.
